### PR TITLE
Vet clinic database: query multiple tables

### PR DESCRIPTION
**Pull request by MozamelJawad**
**In this PR, I worked on the following parts:**

- Create a table named **owners**
- Create a table named **species**
- Modify **animals** table
- Insert data to **owners** table
- Insert data to **species** table

**Write queries (using JOIN) to answer the following 👇 questions:**
- What animals belong to Melody Pond?

<hr>

- List of all animals that are pokemon (their type is Pokemon).

<hr>

- List all owners and their animals, remember to include those that don't own any animal.

<hr>

- How many animals are there per species?

<hr>

- List all Digimon owned by Jennifer Orwell.
3 new
<hr>

- List all animals owned by Dean Winchester that haven't tried to escape.

<hr>

- Who owns the most animals?

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -68,4 +68,33 @@ SELECT AVG(weight_kg) FROM animals;
...
+SELECT full_name, name FROM owners
+left JOIN animals ON owners.id = animals.owner_id;
+SELECT species.name, count(species_id) AS species_count FROM species
+JOIN animals ON species.id = animals.species_id 
+GROUP BY species.name;
+
+SELECT species.name AS species, full_name, animals.name AS animal_name FROM animals 
+JOIN species ON animals.id = species.id 
+JOIN owners ON animals.id = owners.id 
+where full_name = 'Jennifer Orwell';
```
Comment (AbelG101) [inline queries.sql]: - [ ] Oh! you're doing a great job so far but It seems you forgot to filter out "Digimon" species in your query at line `90` as the question is to list all Digimon owned by Jennifer Orwell. 
  - [ ] And also make sure to update the screenshot with the new improved query.😉

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @MozamelJawad 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- New tables are properly added ✔
- All data are properly inserted ✔
- Correct queries for almost all questions ✔
- Correct use of Git Flow ✔
- Descriptive PR title with a top-notch screenshot summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql [hunk 2]
```diff
@@ -68,4 +68,33 @@ SELECT AVG(weight_kg) FROM animals;
...
+left JOIN animals ON owners.id = animals.owner_id;
+SELECT species.name, count(species_id) AS species_count FROM species
+JOIN animals ON species.id = animals.species_id 
+GROUP BY species.name;
+
+SELECT species.name AS species, full_name, animals.name AS animal_name FROM animals 
+JOIN species ON animals.id = species.id 
+JOIN owners ON animals.id = owners.id 
+where full_name = 'Jennifer Orwell'  AND species.name = 'Digimon';
+
```
Comment (thecodechaser) [inline queries.sql:91]: - [ ] Good job updating the query but you missed updating the screenshot in the pull request. Please make sure to update the screenshot in the pull request with the improved query as requested by the previous reviewer.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @MozamelJawad👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight 🎉
- Good job with multiple tables. ✔️
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 2 above)
Comment (MozamelJawad) [reply queries.sql:91]: Thanks for the comment, but the screenshot is already updated.

Comment (MozamelJawad) [reply queries.sql:91]: Oops 😔. I added the screenshot but did not click on the update button to permanently save, I just previewed the changes.
By any way thanks for your review.

**Review by AbayomiOlaoye — APPROVED** [review]
## STATUS: APPROVED :green_circle:  

Hi @MozamelJawad  and @thecodechaser    :wave: :wave:, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

applause

## Highlights
- Added clear screenshots :white_check_mark: 
- Queries look right  ✅ 
- Professional repo documentation ✅ 
- Great teamwork ✅ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**